bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -308,14 +308,13 @@
 
 	<!-- Litre Channel -->
 	<channel-type id="amrLitre">
-		<item-type>Number:Litre</item-type>
+		<item-type>Number:Volume</item-type>
```

Comment:
The label says "litre per second" - so is this rather a flow speed instead of a volume measure? It somehow does not seem to fit together.

Reply:
I just looked at the EEP which uses this channel and it returned a Litre value hence I choose Volume. However the EEP doc also clearly state Litre/Second. So I would indeed change it a flow unit.

@dominikkv Do you own such a device? Is it ok to change it to a flow unit or should we introduce a manufacturer specific EEP?

Reply:
Nope, I do not own such a device. It seems that this channel is used with EEP A5-12-02 and A5-12-03, which are "Automated Meter Reading" for gas & water.

The documentation for the data field MR says: "Cumulative value in m³ or current value in liter/s".

As this binding is using the two channels amrLitre and amrCubicMetre for this, it seems that it is already prefixed with "amr" (= Automated Meter Reading?). So I think it is better to change the unit.

Reply:
I changed to a flow unit now